src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -110,6 +110,10 @@ protected function useTty($input)
      */
     protected function validateEnvironment($site, $environment)
     {
+        // Skip the warning for 'test' and 'live'
+        if (in_array($environment->id, ['test', 'live'])) {
+            return;
+        }
```

Comment:
Please use `$environment->isDevelopment()` for this logic. It'd also be shorter if you ask
```bash
if ($environment->isDevelopment()) {
    $this->validateConnectionMode($environment->get('connection_mode'));
}
```